examples/plot_poisson_2.py

```diff
@@ -80,19 +81,27 @@
 #         \\
 #         \exp(\eta) \sum_i \Big(1 - \frac{\lambda_i}{y_i}\Big)x_{ij}, & \text{if}\ z_i \gt \eta
 #     \end{cases}
+#
+#
+#
+
+########################################################
 
 ########################################################
 
 z = np.linspace(0., 10., 100)
 qu = glm_poissonexp.qu(z)
-plt.plot(z, qu)
+plt.plot(z, qu, label='a')
```

Comment:
can you remove the `plt.hold` in the next line? It's very MATLAB style. You don't need it in python